packages/xo-server/src/xapi-object-to-xo.mjs

```diff
@@ -593,13 +593,21 @@ const TRANSFORMS = {
 
   pif(obj) {
     const metrics = obj.$metrics
+    const isBondMaster = !isEmpty(obj.bond_master_of)
+    const isBondSlave = obj.bond_slave_of !== 'OpaqueRef:NULL'
+
+    // Why is `bond_master_of` a list? Getting the first one in the list seems to be the right way:
+    // https://github.com/xcp-ng/xenadmin/blob/4a9d971dadd04c62f7f77f5ccf1089b4aaa59639/XenModel/XenAPI-Extensions/PIF.cs#L246-L252
+    const bond = isBondMaster ? obj.$bond_master_of[0] : isBondSlave ? obj.$bond_slave_of : undefined
 
     return {
       type: 'PIF',
 
       attached: Boolean(obj.currently_attached),
-      isBondMaster: !isEmpty(obj.bond_master_of),
-      isBondSlave: obj.bond_slave_of !== 'OpaqueRef:NULL',
+      bondMaster: isBondSlave ? link(bond, 'master') : undefined,
```

Comment:
I don't know if `bondMaster`/`bondSlaves` is a good wording since `link(bond, 'master')/link(bond, 'slaves')` will return PIF IDs

Reply:
The logic is that if a PIF is the master of a bond, then `pif.bondSlaves` would give you the "PIF's bond's slaves".

Since we're already on a PIF object, it feels a bit weird to prefix it again with `pif`. So maybe simply `master`/`slaves`? But then we don't know that we're talking about bonds.